core/src/main/java/fr/sncf/osrd/cli/StandaloneSimulationCommand.java

```diff
@@ -0,0 +1,162 @@
+package fr.sncf.osrd.cli;
+
+import com.beust.jcommander.Parameter;
+import fr.sncf.osrd.envelope_sim_infra.EnvelopeTrainPath;
+import fr.sncf.osrd.infra.implementation.signaling.SignalingInfraBuilder;
+import fr.sncf.osrd.infra.implementation.signaling.modules.bal3.BAL3;
+import fr.sncf.osrd.infra_state.implementation.TrainPathBuilder;
+import fr.sncf.osrd.railjson.parser.RJSParser;
+import fr.sncf.osrd.railjson.parser.RJSRollingStockParser;
+import fr.sncf.osrd.railjson.parser.RJSStandaloneTrainScheduleParser;
+import fr.sncf.osrd.railjson.schema.RJSSimulation;
+import fr.sncf.osrd.railjson.schema.rollingstock.RJSLoadingGaugeType;
+import fr.sncf.osrd.railjson.schema.schedule.RJSStandaloneTrainSchedule;
+import fr.sncf.osrd.railjson.schema.schedule.RJSTrainPath;
+import fr.sncf.osrd.railjson.schema.schedule.RJSTrainSchedule;
+import fr.sncf.osrd.reporting.warnings.WarningRecorderImpl;
+import fr.sncf.osrd.standalone_sim.StandaloneSim;
+import fr.sncf.osrd.standalone_sim.result.StandaloneSimResult;
+import fr.sncf.osrd.train.RollingStock;
+import fr.sncf.osrd.train.StandaloneTrainSchedule;
+import java.io.FileWriter;
+import java.io.PrintWriter;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.*;
+
+public class StandaloneSimulationCommand implements CliCommand {
+
+    @Parameter(
+            names = {"--infra_path" },
```

Comment:
If you specify `names` without `--` this option become a main parameter which is better in our case.

You can check the documentation of jcommander here: https://jcommander.org/#_overview
```suggestion
            names = {"infra_path" },
```